kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/net2/configuration/GwtNetInterfaceConfigBuilder.java

```diff
@@ -122,6 +123,9 @@ private void setIpv4Properties() {
         this.gwtConfig.setSubnetMask(this.properties.getIp4Netmask(this.ifName));
         this.gwtConfig.setGateway(this.properties.getIp4Gateway(this.ifName));
         this.gwtConfig.setDnsServers(this.properties.getIp4DnsServers(this.ifName));
+
+        Optional<Integer> mtu = this.properties.getIp4Mtu(this.ifName);
+        this.gwtConfig.setMtu(mtu.isPresent() ? mtu.get() : DEFAULT_MTU);
```

Comment:
I think this can be simplified to:

```suggestion
        this.gwtConfig.setMtu(mtu.orElse(DEFAULT_MTU));
```